node/lib/util/cherry_pick_util.js

```diff
@@ -523,11 +526,47 @@ exports.computeChanges = co.wrap(function *(repo, index, targetCommit) {
 exports.writeConflicts = co.wrap(function *(repo, index, conflicts) {
     let errorMessage = "";
     const names = Object.keys(conflicts).sort();
+    const opener = new Open.Opener(repo, null);
+    const fetcher = yield opener.fetcher();
     for (let name of names) {
-        yield ConflictUtil.addConflict(index, name, conflicts[name]);
+        let conflict = conflicts[name];
+        let configured = false;
+        try {
+            // We just want to see if it's configured
+            const url = yield fetcher.getSubmoduleUrl(name);
+            configured = url !== null;
```

Comment:
There's nothing in the contract of `getSubmoduleUrl` that says it might return a `null`; what's the check for?

Reply:
Well, it definitely happens in the tests...

Reply:
I can't see how it could return a null after a (quick) look at the implementation, but if it can the contract should be updated to explain when; as it is the contract is to throw a `UserError` on failure.

Reply:
if this.d_urls[name] actually contains null, that would happen. I think this may only happen in tests.  I could change the code to instead detect that case and throw UserError (then catch it here instead of checking for null) -- what do you think?

Reply:
I'd want to understand how `this.d_urls` comes to have null values first, and what it means.

Reply:
Oh, wait a second -- I think I just checked the wrong thing.  In fact it's not null. I could just set configured to true here.